content/en/docs/advanced-tasks/configuring-olm.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Configuring OLM"
+linkTitle: "Configuring OLM"
+weight: 3
+---
+
+# Configuring OLM
+
+The [Operator-lifecycle-Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager) controller can be configured through an  [OLMConfig CustomResourceDefinition (CRD)](https://github.com/operator-framework/operator-lifecycle-manager/blob/3f3f00644ca4a50d845db2b97c12afcea56fd5d8/deploy/chart/crds/0000_50_olm_00-olmconfigs.crd.yaml) named `cluster`. This document will outline what configurations OLM currently supports.
+## Disabling Copied CSVs
```

Comment:
nit/feel free to ignore: add a newline between line 9 and 10 to suppress any markdown linting warnings?

Reply:
Done.